x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/hooks/use_private_location_permission.ts

```diff
@@ -10,16 +10,17 @@ import { ClientPluginsStart } from '../../../../plugin';
 import { BrowserFields, ConfigKey } from '../../../../../common/runtime_types';
 
 export function usePrivateLocationPermissions(monitor?: BrowserFields) {
-  const kServices = useKibana<ClientPluginsStart>().services;
+  const { fleet } = useKibana<ClientPluginsStart>().services;
 
-  const canSaveIntegrations: boolean =
-    !!kServices?.fleet?.authz.integrations.writeIntegrationPolicies;
+  const canSaveIntegrations: boolean = !!fleet?.authz.integrations.writeIntegrationPolicies;
```

Comment:
Can you move this next to the other permission variable since they are related business logic? Also, can we, at least in this file, decide on using either `!!` or `Boolean`. it is going to drive me crazy seeing both right next to each other.